Automattic/woocommerce-payments
# Use lint-staged for husky pre-commit hook

#### Changes proposed in this Pull Request

* Lint staged js, jsx, scss and php files before the commit

#### Testing instructions

* Try to commit js, jsx, scss and php files which break lint rules. The commit should fail with lint errors displayed.
* Fix lint errors and try to commit again. Commit should succeed.
* Reset committed changes

-------------------
